vengine/vengine/physics/CollisionDispatcher.cpp

```diff
@@ -0,0 +1,33 @@
+#include "CollisionDispatcher.h"
+
+btPersistentManifold* CollisionDispatcher::getNewManifold(const btCollisionObject* b0, const btCollisionObject* b1)
+{
+	btPersistentManifold* man = btCollisionDispatcher::getNewManifold(b0, b1);
+
+	int num = this->getNumManifolds();
+	this->types.resize(num);
+	this->types[(size_t)num - 1] = CallbackType::ENTER;
```

Comment:
Can num ever be 0? If it can, then this code should handle that case. If not, then no change is needed.